app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,37 @@
+class FoodsController < ApplicationController
+  # before_action :set_recipe, only: %i[show edit update destroy]
```

Comment:
**You guys did really great job!**

- [ ] The only thing left is cleaning. Let me point you out to those places. For starting, please remove line 2 in this file.